### PR TITLE
Adding @wasade's parallel to per sample fasta

### PR DESCRIPTION
**Pull request by josenavas**

### CODE REVIEW

**Comment by josenavas** [comment]
Hang on a second - I just noticed that with a small change the user can specify if he wants fasta or fastq format (it may be useful for EBI submission)

**Comment by josenavas** [comment]
Awesome this is ready for review!

File: qiita_files/demux.py [hunk 1]
```diff
@@ -455,6 +461,60 @@ def to_per_sample_ascii(demux, samples=None):
...
+        Path to output directory to store the per sample fasta.
+        Defaults to current directory
+    n_jobs : int, optional
+        Number of jobs to run in parallel. Defaults to 1
+    out_format : {'fastq', 'fasta'}
+        The format in which the output files should be written.
+    """
+    if out_format == 'fastq':
+        formatter = format_fastq_record
+        file_name_fmt = "%s.fq"
```
Comment (antgonza) [inline qiita_files/demux.py]: why not fastq?

Comment (josenavas) [inline qiita_files/demux.py]: I saw that in the tests 'fq' was used, that's why I used that, but no strong feelings so I can change.

**Comment by josenavas** [comment]
@antgonza any clue on why coveralls is not working here?

**Comment by antgonza** [comment]
nop, could you use the new structure that we have been using with other repos?

**Comment by josenavas** [comment]
Done - can you make sure that coveralls is enabled on this repo? I can't check on my coveralls page

**Comment by antgonza** [comment]
I did and is on.

**Comment by antgonza** [comment]
Anyway, from the log:
```
TOTAL                             629     47    93%
```
Thus, merging.

**Comment by wasade** [comment]
bit late to the game but :+1: :)

**Pull request merged**